templates/tutorialv2/includes/tags_authors.part.html

```diff
@@ -69,8 +69,8 @@
         </p>
 
         <p>
-            {% if online and reading_time and not is_part_or_chapter %}
-                {% trans "Temps de lecture estimé : " %}{{ reading_time|minute_to_duration }}
+            {% if online and not is_part_or_chapter %}
+                {% trans "Temps de lecture estimé : " %}{{ reading_time|humanize_duration }}
             {% elif online and not is_part_or_chapter %}
                 {% trans "Temps de lecture estimé à moins d’une minute." %}
             {% endif %}
```

Comment:
Cela devrait être possible étant donné que la fonction fonctionne correctement quand `reading_time` est nul. On peut même inclure la balise `<p>` dans la condition `if`.

```suggestion
            {% if online and not is_part_or_chapter %}
                {% trans "Temps de lecture estimé à " %}{{ reading_time|humanize_duration }}
            {% endif %}
```

Reply:
Ouais, j'avais retiré un bout de condition en gérant correctement le cas nul, mais j'avais pas pensé à regarder autour plus que ça. :D Je fais une modif en ce sens.